### DISCUSSION TITLE
TESTCONTAINERS_RYUK_DISABLED not applied, possible regression?

### DISCUSSION THREAD

**Discussion by einarwar** [testcontainers/testcontainers-go#2673] [Q&A]
Hi

Since I use testcontainers-go in a CI environment, i need to disable ryuk. On version 0.31.0 i set the environment variable `TESTCONTAINERS_RYUK_DISABLED=true`, and ryuk was disabled. However, when trying to upgrade to 0.32.0, it does not seem like ryuk is disabled any more. When i try to run my tests, i get the error `Failed to start docker-compose:  failed to connect to reaper: dial tcp 192.168.5.1:32810: connect: connection refused: Connecting to Ryuk on 192.168.5.1:32810 failed`, which indicates that ryuk is in fact not disabled. 

I have not been able to try to check if setting it in a `.testcontainers.properties`, since this is run in a CI environment and cannot rely on files outside the git repository.

**Comment by mdelapenya** [comment]
Hi @einarwar thanks for reporting this. I've tried to reproduce this and I'm not able to reproduce it yet:

- I ran the compose module tests
- I ran the tests in https://github.com/mdelapenya/testcontainers-go-compose (an example repo I use for testing compose releases in the wild)

In both cases, setting the variable to `true` disabled Ryuk:

```
TESTCONTAINERS_RYUK_DISABLED=true go test ./... -v -count=1
```

I verified it in a separate terminal with `watch -n 0.1 docker ps`

**Comment by einarwar** [comment]
Hi @mdelapenya and thanks for your reply.

We later tried to upgrade from 0.31.0 to 0.33.0 and the problem was no longer present.
End of the story we were able to upgrade :)

**Reply by mdelapenya** [reply]
Fantastic!!!! Super glad you fixed it
